ztypes_netbsd_32bit_int.go

```diff
@@ -0,0 +1,13 @@
+// +build openbsd
+// +build 386 amd64 arm
```

Comment:
What about arm64? Is that a thing on netbsd?

Reply:
Yes, it is, I can add it as well.